toolchain/lower/file_context.h

```diff
@@ -157,6 +188,50 @@ class FileContext {
 
   auto BuildVtable(const SemIR::Class& class_info) -> llvm::GlobalVariable*;
 
+  // Function is used to track which specifics were lowered for each generic.
+  // These are added one by one while lowering their definitions.
+  auto AddLoweredSpecificForGeneric(SemIR::GenericId generic_id,
+                                    SemIR::SpecificId specific_id) {
+    lowered_specifics_[generic_id.index].push_back(specific_id);
+  }
+  // Initialize a return a SpecificFunctionFingerprint* instance for specific.
```

Comment:
WDYT of adding blank lines between functions, so that it's easier to see where things start and end?